samples/DLLMethodCaller/DLLMethodCallerEA.cs

```diff
@@ -7,29 +7,36 @@ namespace DLLMethodCaller
 {
     public class DLLMethodCallerEA : MqlApi
     {
+        [ExternVariable]
         public string ActionName { get; set; }
 
         // GiveMoney action
+        //[ExternVariable]
         public int MoneyAmount { get; set; }
-
         // PredictFuture action
+        //[ExternVariable]
         public DateTime MyBirthday { get; set; }
+        //[ExternVariable]
         public bool AmMarried { get; set; }
+        //[ExternVariable]
```

Comment:
ExternVariable attribute is optional. It should work without it if the variable is public. You can leave it out or keep it if you want, but not leave it commented.